spring-r/src/main/resources/plot.R

```diff
@@ -1,3 +1,44 @@
+ #
+ # Copyright (c) 2012, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
In the R source files, you don't need an opening # and closing `#/` tags. Remove.